README.md

```diff
@@ -111,6 +112,13 @@ await client.pass('someUserId');
 await client.superLike('someUserId');
 ```
 
+### `getUpdates`
+
+```javascript
+await client.getUpdates();
```

Comment:
Can you add comments / distinguish that the first function gets all updates from account creation and the second gets all updates from the specified `ISO-8601` timestamp?